package.json

```diff
@@ -51,7 +51,7 @@
     }
   },
   "scripts": {
-    "start": "./cli/edgemicro start",
+    "start": "node app.js",
```

Comment:
why would it break other? "npm start" is the command anyone uses. this has the same effect.